src/main/java/http/helper/Handler.java

```diff
@@ -0,0 +1,9 @@
+package http.helper;
```

Comment:
small feedback - the `helpers` and `utils` directories often end up being a "dumping ground" in projects. I'm not referring to your project here, but just in general - it's easy to put files there as almost everything can match the description of "helper" or "util". This is happening a lot at FLA and to prevent making it worse, we keep a close eye each time one of us tries to put something there 👁️ 

`Handler` to see seems like an almost "equal" element of your structure to your router or parser, maybe it should have its own directory too? Even though it's small when it comes to the number of lines. 

The helpers or utils are better for things like eg. general rules for string transformation, some dates handling that are shared by the whole project, logic that decides if something is empty of not, code snippets shared by multiple classes etc.